styleguide/derek/examples/homepage.ejs

```diff
@@ -303,11 +303,11 @@
                 </div>
               </div>
             </div>
-            <div class="col-md-6 col-md-push-2">
```

Comment:
I removed this `.col-md-push-2` to fix the extra 2-column push beyond the 12 units that were already covered by the two `.col-md-6` divs.